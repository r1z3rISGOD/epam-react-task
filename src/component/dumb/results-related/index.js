import {ResultsRelated} from "./results-related"
export {ResultsRelated}
