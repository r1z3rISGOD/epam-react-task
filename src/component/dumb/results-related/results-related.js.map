{"version":3,"file":"results-related.js","sourceRoot":"","sources":["results-related.tsx"],"names":[],"mappings":";;;AAAA,6BAA8B;AAC9B,kCAA+B;AAMxB,IAAM,cAAc,GAAmC,UAAC,EAAoB;QAAlB,gBAAgB,sBAAA;IAC/E,OAAO,CACD,8BAAM,SAAS,EAAC,iBAAiB;;QAAW,gBAAgB;iBAAc,CAC/E,CAAA;AACH,CAAC,CAAA;AAJY,QAAA,cAAc,kBAI1B"}