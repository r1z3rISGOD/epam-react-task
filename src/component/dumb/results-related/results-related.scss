@import "./src/_variables/variables.scss";

.results-related  {
  font-family: $font;
  font-size: 1.5rem;
}
