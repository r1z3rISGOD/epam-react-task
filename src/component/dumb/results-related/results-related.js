import React from 'react'
import './results-related.scss'
import PropTypes from 'prop-types'

export const ResultsRelated = (props) => {
  const { openedFilmsGenre } = props
  return (
        <span className="results-related">Films by {openedFilmsGenre} genre</span>
  )
}

ResultsRelated.propTypes = {
  openedFilmsGenre: PropTypes.string
}
