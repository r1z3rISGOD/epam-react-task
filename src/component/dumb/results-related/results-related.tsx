import * as React from 'react'
import './results-related.scss'

interface ResultsRelatedProps {
  openedFilmsGenre: string
}

export const ResultsRelated : React.FC<ResultsRelatedProps> = ({ openedFilmsGenre }) => {
  return (
        <span className="results-related">Films by {openedFilmsGenre} genre</span>
  )
}
