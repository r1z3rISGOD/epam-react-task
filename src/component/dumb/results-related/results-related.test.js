import { ResultsRelated } from './results-related.tsx'
import React from 'react'

it('results related', () => {
  const comp = shallow(<ResultsRelated/>)
  expect(comp).toMatchSnapshot()
})
