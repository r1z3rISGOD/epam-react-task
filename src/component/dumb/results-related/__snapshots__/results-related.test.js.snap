// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`results related 1`] = `
<span
  className="results-related"
>
  Films by 
   genre
</span>
`;
